lib/constants/refNumberType.js

```diff
@@ -0,0 +1,12 @@
+const REF_NUMBER_TYPE = {
+  continuationRefNumber: 'Vendor continuation reference number',
```

Comment:
just curious, these values are used on BE?

Reply:
It's enum list https://github.com/folio-org/acq-models/pull/306/files#diff-79468c03bd8bff6702cf8872d5ff045180754fb82205d8d8edee1b5f4f216147R13

Reply:
:)